recipes-extended/rsyslog/rsyslog_8.2206.0.bbappend

```
@@ -0,0 +1,16 @@
+PR = "r1"
+
+# We don't want silly logrotate deps
+# As no one is currently using it
+# The same is valid for systemd service
+# as we provide our-own
+RDEPENDS:${PN}:remove = "logrotate"
+RCONFLICTS:${PN} = ""
+SYSTEMD_SERVICE:${PN} = ""
+
+# Removing some not needed files
+# (logrotate, conf and systemd services)
+do_install:append() {
+  rm -rf "${D}/etc" \
+         "${D}/lib"
+}
```

```suggestion
do_install:append() {
  rm -rf "${D}${sysconfdir}" \
         "${D}/lib"
}
```
Just minor thing to make it prettier. For `${D}/lib` is annoying that replacing with `${D}${systemd_unitdir}` will not work since there will be a dangling `${D}/lib` (as the systemd variable refers to one level deeper).